### PR TITLE
747 740 withdrawn

### LINKED ISSUE #740: When I upload withdrawn records with missing info, I see a message telling me where I can find them
When I batch upload withdrawn records with missing info, I won't know where to find them since they don't appear on the active roster. 

This issue doesn't happen when withdrawing records through the app, as all info is required. 

Screen Shot 2020-11-11 at 3 22 45 PM

**Only change the text if withdrawn records have missing info**

### LINKED ISSUE #747: When I don't have any withdrawn records, I see placeholder text telling me that.
Replace the "no records in your roster" alert with:
Screen Shot 2020-11-12 at 9 25 54 AM

### PR DESCRIPTION
**Pull request by mmazanec22**
Closes #747 closes #740 
Also fixes a bug where the backend was incorrectly parsing withdrawn param and only ever returning withdrawn enrollments.

### CODE REVIEW

**Comment by mmazanec22** [comment]
This needs to be rewritten bc the alert has to count all children the user has access to

File: client/src/containers/Roster/tableColumns.tsx [hunk 1]
```diff
@@ -31,17 +31,17 @@ export const tableColumns: (
     {
       className: tableColumnClassName,
       name: ColumnNames.NAME,
-      sort: (row) => row.lastName || '',
+      sort: (row) => row?.lastName || '',
```
Comment (mmazanec22) [inline client/src/containers/Roster/tableColumns.tsx:34]: For some reason axe was complaining about these being undefined-- idk why but this made it happy

File: client/src/utils/models/getLastEnrollment.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { Child } from '../../shared/models';
+import moment from 'moment';
+import { propertyDateSorter } from '../dateSorter';
+
+export const getLastEnrollment = (child: Child) => {
+  return (child.enrollments || []).sort((a, b) =>
```
Comment (jlhogan) [inline client/src/utils/models/getLastEnrollment.ts:6]: these enrollments should already be filtered coming back from the server. So really both this and "getcurrentenrollment" could probably be collapsed into a `getMostRecentEnrollment` which just pulls the last thing off the array.

**Review by jlhogan — APPROVED** [review]

**Review by bamader — APPROVED** [review]
LGTM!

**Pull request merged**